config/config.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	"errors"
+	"flag"
+	"strings"
+)
+
+type Config struct {
+	PolicyId  string
+	Tls       *TlsConfig
+	Port      int
+	RodeHost  string
```

Comment:
In the collectors, we've usually made a `RodeConfig` struct that also has an `insecure` setting controlling TLS use.